dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/NeuralNetworkGroupCommDriver.java

```diff
@@ -132,7 +132,8 @@ public void onNext(final ActiveContext activeContext) {
         LOG.log(Level.FINEST, "Submit group comm parameter server context {0} to {1}",
             new Object[]{nnCtrlCtxtId, contextId});
 
-        activeContext.submitContextAndService(groupCommContextConf, groupCommServiceConf);
+        activeContext.submitContextAndService(groupCommContextConf,
+            Configurations.merge(groupCommServiceConf, neuralNetworkESParameters.getServiceConfiguration()));
```

Comment:
This change is for binding a matrix factory implementation which is used in the codecs for serializing and de-serializing `Matrix`.